### DISCUSSION TITLE
combi mode : add separator between `windowcd` and `drun` entries ?

### DISCUSSION THREAD

**Discussion by fenugrec** [davatorium/rofi#1956] [Theming] [answered]
First: rofi has been awesome so far, great improvement to my workflow.

I'm trying to tweak a theme (builtin `glue_pro_blue`) so that my usual command, `rofi -show combi -combi-modes windowcd,drun` , visually separates the entries for windowcd and drun - a simple horizontal line would do.

The obvious , modifying `listview { border:....} ` in my config.rasi doesn't work as expected since the whole thing is one single listview, not one per mode as I hoped.

Any suggestions to achieve this ?
Thanks !

**Comment by DaveDavenport** [comment, marked as answer]
this is not possible.

**Reply by fenugrec** [reply]
Unfortunate. Thanks for confirming though.

Best I could come up with to improve visual contrast between windowcd and drun entries : make drun entries all italics, and vertically align the names.
```
	drun-display-format: "[<span weight='light' size='small'><i>{name}  ({generic})</i></span>]";
	drun { display-name: "drun     ";}
```
![combi](https://github.com/davatorium/rofi/assets/10098613/8909b217-2742-4f98-acb2-44f77d04d1a0)
